pkg/acsengine/const.go

```diff
@@ -41,8 +41,8 @@ const (
 )
 
 const (
-	KubernetesHyperkubeImageName         = "hyperkube-amd64:v1.5.3"
-	KubernetesDashboardImageName         = "kubernetes-dashboard-amd64:v1.5.1"
+	KubernetesHyperkubeImageName         = "hyperkube-amd64:v1.6.0"
```

Comment:
@JackQuincy , its changing to kubernetes-dashboard-amd64:**v1.6.0** from **v1.5.1**. Does 1.6.0 also have that bug? If not then we should take this change.